web/src/app/services/navigation/navigation.service.ts

```diff
@@ -181,6 +183,18 @@ export class NavigationService implements OnDestroy {
     this.router.navigate([SURVEY_SEGMENT, SURVEY_ID_NEW]);
   }
 
+  async getAccessDeniedLink(): Promise<string | undefined> {
+    const accessDeniedMessage = await this.dataStore.getAccessDeniedMessage();
```

Comment:
Where do we document the contents of the DB config? Can we create a wiki page or readme for this? Ok to do after merging as well.